microservices/scraper-{{cookiecutter.project_slug}}/app/scraper.py

```diff
@@ -0,0 +1,15 @@
+from bs4 import BeautifulSoup
+import requests
+import json
+
+
+"""
```

Comment:
Put the doc strings on one line, similar to https://github.com/hack4impact/flask-base/blob/master/app/utils.py, it just looks a little cleaner